components/Accordion/Accordion.jsx

```diff
@@ -99,13 +105,46 @@ const AccordionContent = styled.div`
 `;
 
 const StyledIcon = styled(Icon)`
-  ${({ theme: { baseFontSize } }) => `
+  ${({
+    theme: {
+      baseFontSize,
+      spacing: { large },
+      colors: { primary },
+    },
+  }) => `
     font-size: ${baseFontSize * 1.5}px;
+    margin-right: ${large}px;
+    color: ${primary[700]};
   `}
+
   line-height: inherit;
 `;
 
-AccordionItem.displayName = 'AccordionItem';
+const AccordionItem = styled.li`
+  ${({
+    theme: {
+      colors: { primary },
+      spacing: { medium },
+    },
+    opened,
+  }) => `
+  &:first-child {
+    border-radius: ${opened ? '4px' : '4px 4px 0px 0px'};
+  }
+
+  &:last-child {
+    border-radius: ${opened ? '4px' : '0px 0px 4px 4px'};
+    border-bottom: none;
+  }
+
+  border-left: ${opened ? `4px solid ${primary[700]}` : `inherit`};
```

Comment:
I think when the state is closed border left should be same, but with background color. Currently the header jumps a bit for the right when the state changes.